web_ui/src/pages/project-details/components/project-test/test-media-items-list.component.tsx

```diff
@@ -34,6 +35,23 @@ interface TestMediaItemsListProps {
     selectedLabelId?: string;
 }
 
+const viewModeSettings = {
+    [ViewModes.SMALL]: { minItemSize: 70, gap: 4, maxColumns: 6 },
+    [ViewModes.MEDIUM]: { minItemSize: 90, gap: 4, maxColumns: 4 },
+    [ViewModes.LARGE]: { minItemSize: 100, gap: 4, maxColumns: 2 },
+    [ViewModes.DETAILS]: { size: 85, gap: 0 },
+};
+
+const getTestMediaItemId = (item: TestMediaItem) => {
+    const annotationId = 'testResult' in item ? item.testResult.annotationId : '';
+
+    if (item.type === MEDIA_TYPE.IMAGE) {
+        return `${item.media.identifier.imageId}-${annotationId}`;
+    }
+
+    return `${item.media.identifier.videoId}-${annotationId}`;
```

Comment:
I'm wondering if this should not also check for a video frame number. Though I guess the `annotationId` will make the id unique so this isn't a blocker.